packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -257,6 +256,9 @@ export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
     }),
   };
 
+  // Excluding `inversed` prop from `<div>` label because it's not a valid attr
+  const { inversed, ...divLabelProps } = labelProps;
```

Comment:
For some reason I'm still getting the same error when I run the unit tests:

```
console.error
      Warning: Received `true` for a non-boolean attribute `inversed`.
      
      If you want to write it to the DOM, pass a string instead: inversed="true" or inversed={value.toString()}.
          at button
          at div
          at Dropdown (/Users/patrickwolfert/Projects/design-system/packages/design-system/src/components/Dropdown/Dropdown.tsx:139:16)
```

Does the error go away for you?

Reply:
I ran `yarn test` locally and everything passed. When I built locally, the `inversed` attr was removed so I assumed it was fixed

Reply:
Wait! Maybe this is a web component issue? I need to investigate that when I sign on tomorrow.

Reply:
J/k, I think I've fixed it for real this time. Wasn't a 🚾 issue

Reply:
Nice, that fixes it 👍 

Alternatively, it might be clearer what is going on if you leave the first one as `inversed,` and change this second one to something like

```suggestion
  const { inversed: _removedInversed, ...divLabelProps } = labelProps;
```